plugins/win-dshow/CMakeLists.txt

```diff
@@ -1,150 +1,175 @@
 if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/libdshowcapture/dshowcapture.hpp")
-	message(STATUS "libdshowcapture submodule not found!  Please fetch submodules.  win-dshow plugin disabled.")
-	return()
+  message(
+    FATAL_ERROR
+      "OBS:   libdshowcapture submodule not found!  Please fetch submodules. win-dshow plugin disabled."
+  )
+  return()
+endif()
+
+option(ENABLE_VIRTUALCAM "Enable building with VirtualCam (Windows)" ON)
```

Comment:
Nitpick:  VirtualCam here, but Virtualcam below.  Use "Virtual Camera" if possible, here and throughout.  If "Virtual Camera" is not possible, then use "VirtualCam".

Reply:
Adapted.